src/main/resources/User-Registrations.json

```
@@ -13,7 +13,9 @@
       "counter": 0,
       "encryptionScheme": "salted-pbkdf2-hmac-sha256",
       "factor": 1,
-      "debug": true
+      "debug": true,
+      "numberOfApplications": 1000,
```

Defaults in JSON files are 10 tenants and 100 applications. I think this should match.
```suggestion
      "numberOfApplications": 100,
```